src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -60,6 +60,7 @@
   grid-column: 3/4;
   position: relative;
   top: 2px;
+  cursor: pointer;
```

Comment:
I'm surprised. How home you have something for entity viewer in this PR? 😄

Reply:
I once merged the layout update branch before it got merged into dev. That could be why this got left out. I think I need to remove this since it is shown as a difference.

Reply:
That's done